docs/user-guide.md

```diff
@@ -139,3 +142,42 @@ kubectl create -f examples/volume/local-pv.yaml
 ```
 kubectl create -f examples/volume/pod-local.yaml
 ```
+
+#### NFS volume
+* create the NFS server
+```
+kubectl create -f nfs-server.yaml
+```
+* get the IP address for the NFS server and update the pod yaml
+```
+kubectl get services
+vi pod-nfs.yaml
+```
+* create the pod that uses the volume
+```
+kubectl create -f pod-nfs.yaml
+```
+
+#### EBS volume
+The Volume for the first two examples must be created before you can run the examples. The `VolumeId` must be updated in the yaml files to get this to work.
+To create a volume you can use the command line or web UI:
+```
+aws ec2 create-volume --volume-type gp2 --size 10 --availability-zone us-west-1
+```
+The `VolumeId` is part of the returned information of the create command.
+
+* create the pod that uses a direct volume reference:
+```
+kubectl create -f pod-ebs-direct.yaml
+```
+* create the volume (pv) and a pod that uses a pvc to claim the existing volume:
```

Comment:
volume -> persistent volume
a pvc -> persistent volume claim (pvc)

it will also be good if we can link to Kubernetes PV/PVC document for them

Reply:
updated